### PR TITLE
[STAKAN-111] Add Landing Page

### PR DESCRIPTION
**Pull request by Kud8**
Hello, brave developer, nice day to merge a pull request, ain't it?

## Follow the checklist to proceed

- [ ] I've performed a code self-review
- [ ] I've tested feature or bugfix on my own
- [ ] If I make changes to another person's module, I've asked how to use it or request a review
- [ ] I've checked mobile & tablet screens (if have changes in UI)
- [ ] Local build successfully completed
- [ ] Branch name and pull request title are according to [the requirements](https://ankrnetwork.atlassian.net/wiki/spaces/STAKAN/pages/1666646162/Branches+and+pull+request+naming)

### CODE REVIEW

File: src/modules/landing/components/App/App.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import FAQ from '../FAQ/FAQ';
+import BgVideo from '../BgVideo/BgVideo';
+import Menu from '../Menu/Menu';
+
+const App = () => {
+  const classes = useStyles();
+  const [isMenuOpened, setIsMenuOpened] = useState(false);
+
+  const handleMenuToggle = () => {
+    setIsMenuOpened(v => !v);
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/App/App.tsx:22]:
```suggestion
    setIsMenuOpened(isMenuOpened => !isMenuOpened);
```

plz make ur code more readable for others

File: src/modules/landing/components/App/App.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  };
+  const handleMenuClose = () => {
+    setIsMenuOpened(false);
+  };
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/App/App.tsx:26]:
```suggestion

```
toggle should be enough

**Comment by AndrewKhizhnyak** [comment]
Why is all text not in en-us? Don't we want to support i18n here?

File: src/modules/landing/components/FAQ/FAQ.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  {
+    question: 'Why should I get a Parachain bond?',
+    answer: (
+      <span>
+        <u>Liquidity</u>: Parachain bonds enable liquidity on bonded DOT through
+        a dedicated auction marketplace on Bounce Finance.
+        <br />
+        <br />
+        <u>High Yield</u>: Furthermore, it integrates rewards from projects
+        winning Parachain Slot Auctions, meaning that Parachain Bonds are like
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/FAQ/FAQ.tsx:47]: i18n

File: src/modules/landing/components/FAQItem/FAQ.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  answer: React.ReactElement;
+}
+
+const FOLDING_TIME = 300;
+
+const FAQItem = ({ question, answer }: FAQItemProps) => {
+  const classes = useStyles();
+  const [isOpened, setIsOpened] = useState(false);
+
+  const handleToggleAnswer = () => setIsOpened(v => !v);
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/FAQItem/FAQ.tsx:20]:
```suggestion
  const handleToggleAnswer = () => setIsOpened(!isOpened);
```

File: src/modules/landing/components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+import { useStyles } from './Styles';
+
+const Footer = () => {
+  const classes = useStyles();
+  const isDesktop = useMediaQuery('(min-width:960px)');
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.rights}>
+        ©2021 Ankr Staking. All rights reserved
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/Footer/Footer.tsx:13]: i18n

File: src/modules/landing/components/Info/Info.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+          <span>&nbsp;DOT by Ankr</span>
+        </InfoItem>
+
+        <InfoItem value={1233}>
+          Number of listed projects on Ankr Crowdloan marketplace
+        </InfoItem>
+
+        <InfoItem value={123456} startValue={100000}>
+          Estimated USD amount of rewards to be distributed by Ankr Multisig
+          Escrow Wallet
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/Info/Info.tsx:23]: i18n

File: src/modules/landing/components/InfoItem/InfoItem.tsx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+              <div className={classes.value}>
+                <AnimatedNumber
+                  animateToNumber={value}
+                  includeComma
+                  config={{ tension: 150, friction: 40 }}
+                />
+              </div>
+              <div className={classes.description}>{children}</div>
+            </>
+          ) : null}
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/InfoItem/InfoItem.tsx:39]:
```suggestion
          {isShow && (
            <>
              <div className={classes.value}>
                <AnimatedNumber
                  animateToNumber={value}
                  includeComma
                  config={{ tension: 150, friction: 40 }}
                />
              </div>
              <div className={classes.description}>{children}</div>
            </>
          )}
```

File: src/modules/landing/components/Menu/Menu.tsx [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+  onClose: () => void;
+  isVisible: boolean;
+}
+const Menu = ({ onClose, isVisible }: MenuProps) => {
+  const classes = useStyles();
+  const isMobile = useMediaQuery('(max-width:600px)');
+
+  const handleClose = (e: React.MouseEvent<HTMLDivElement>) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/Menu/Menu.tsx:17]: why ignore?

File: src/modules/landing/components/SectionWrapper/SectionWrapper.tsx [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+  children,
+  needTimeout = true,
+  delay = 100,
+}: SectionWrapperProps) => {
+  const [isShow, setIsShow] = useState(false);
+  const { ref, inView } = useInView();
+
+  useEffect(() => {
+    if (inView) {
+      if (needTimeout === true) {
```
Comment (AndrewKhizhnyak) [inline src/modules/landing/components/SectionWrapper/SectionWrapper.tsx:20]:
```suggestion
      if (needTimeout) {
```

**Review by AndrewKhizhnyak — CHANGES_REQUESTED** [review]
need changes

**Comment by Kud8** [comment]
> Why is all text not in en-us? Don't we want to support i18n here?

No

File: src/modules/landing/components/Info/Info.tsx (hunk 6 above)
Comment (Kud8) [reply src/modules/landing/components/Info/Info.tsx:23]: We don't use it here

File: src/modules/landing/components/Footer/Footer.tsx (hunk 5 above)
Comment (Kud8) [reply src/modules/landing/components/Footer/Footer.tsx:13]: We don't use it here

File: src/modules/landing/components/FAQItem/FAQ.tsx (hunk 4 above)
Comment (Kud8) [reply src/modules/landing/components/FAQItem/FAQ.tsx:20]: setState is asynchronous so we should use callback in toggle - it is best practice

File: src/modules/landing/components/App/App.tsx (hunk 1 above)
Comment (Kud8) [reply src/modules/landing/components/App/App.tsx:22]: setState is asynchronous so we should use callback in toggle - it is best practice

**Review by AndrewKhizhnyak — APPROVED** [review]

**Pull request merged**